src/components/JobsBoard/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+
+export default function JobBoard({className}) {
+    return <>
+      <div id="recruitee-careers" className={className}/>
+      <Helmet>
+        <script type="text/javascript">{`var rtscript = document.createElement('script');  rtscript.type = 'text/javascript';  rtscript.onload = function() { var widget = new RTWidget({
+          "companies": [
+            38057
```

Comment:
Should this be configurable? At least via ENV var?

Reply:
Ideally, they should, but this is a JSON of 18 settings and its unlikely any of these will be changed without wanting to test them locally. Will add them if that turns out not to be the case.